Gemfile.lock

```diff
@@ -73,13 +73,13 @@ GEM
       msgpack (~> 1.0)
     builder (3.2.3)
     byebug (10.0.2)
-    capybara (3.1.0)
+    capybara (3.2.1)
```

Comment:
Always be careful when you update your gems, it could break with the normal behavior of your app!